mysql-test/suite/galera_3nodes/t/galera_ist_gcache_rollover2.test

```diff
@@ -0,0 +1,106 @@
+#
+# Test that if the gcache rolls over while IST is in progress, IST will still complete.
+# This is achieved by using the ist_sender_send_after_get_buffers Galera dbug sync point to block the donor after 
+# the first gcache buffer has been locked for IST.
+#
+# After IST blocks, we roll over the gcache and resume IST to confirm that it completes successfully.
+#
+# Two nodes perform IST at the same time in order to make the test more stressfull
+#
+# This test is galera_ist_gcache_rollover.test with applied patch attached to the Galera upstream ticket https://github.com/codership/galera/issues/385
+
+--source include/galera_cluster_3nodes.inc
+--source include/have_debug_sync.inc
+--source suite/galera/include/galera_have_debug_sync.inc
+
+#--let $galera_connection_name = node_3
+#--let $galera_server_number = 3
+#--source include/galera_connect.inc
```

Comment:
Commented out code, please re-add or remove.

Reply:
My intention here was to just apply mentioned patch (https://github.com/codership/galera/issues/385) to galera_ist_gcache_rollover.test. So these 2 test files differ like this:
![image](https://user-images.githubusercontent.com/50845928/84995679-9560c780-b14c-11ea-9ec3-488b0d41d3e4.png)
This is mentioned at the top of the file as the last line of comments.

All problems mentioned here are just inherited from original test. Do you think we should rework this test? I'm not sure...